lib/config/src/package/package_hash.rs

```diff
@@ -4,29 +4,35 @@ use crate::{hash::Sha256Hash, package::PackageParseError};
 ///
 /// Currently only supports the format: `sha256:<hash>`.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct PackageHash(Sha256Hash);
+pub enum PackageHash {
+    Sha256(Sha256Hash),
+}
 
 impl PackageHash {
     const STR_PREFIX: &'static str = "sha256:";
```

Comment:
This should probably be called `ShA256_STR_PREFIX`.

Reply:
Shouldn't enum variants be camelcase.

Reply:
I meant the string:
```
const STR_PREFIX: &'static str = "sha256:";
```
should be:
```
const SHA256_STR_PREFIX: &'static str = "sha256:";
```

Reply:
@PappuPanaut Please revert changes on the enum variant and instead fix the const str name.